evalml/utils/woodwork_utils.py

```diff
@@ -110,13 +111,20 @@ def _retain_custom_types_and_initalize_woodwork(
         return ww.init_series(new_dataframe, old_logical_types)
     if ltypes_to_ignore is None:
```

Comment:
@freddyaboulton do we have a plan for phasing out `_retain_custom_types_and_initialize_woodwork`?  I know your PR dropped some usages of it.  If we need to file an issue with WW to move some functionality over, I'll do it.

Reply:
@chukarsten I think we need it for the time being for the Imputer because the imputer will "break" the schema and we need a way of recreating it while allowing some of the types to change. I've been told that https://github.com/alteryx/woodwork/issues/965 will help with that but I think we need to think about that further.